master/getting-started/kubernetes/installation/config-options.md

```diff
@@ -67,23 +67,79 @@ The following table outlines the supported `ConfigMap` options for etcd:
 
 | Option                 | Description    | Default
 |------------------------|----------------|----------
-| etcd_endpoints         | A comma separated list of etcd nodes. | http://127.0.0.1:2379
-| etcd_ca                | The location of the CA mounted in the pods deployed by the DaemonSet. | None
-| etcd_key               | The location of the client cert mounted in the pods deployed by the DaemonSet. | None
-| etcd_cert              | The location of the client key mounted in the pods deployed by the DaemonSet. | None
+| etcd_endpoints         | Comma-delimited list of etcd endpoints to connect to. | http://127.0.0.1:2379
+| etcd_ca                | The root certificate of the CA that issued the etcd server certificate. Configures `{{site.nodecontainer}}`, the CNI plugin, and the Kubernetes controllers to trust the signature on the certificates provided by the etcd server. | None
+| etcd_key               | The private key of the `{{site.nodecontainer}}`, the CNI plugin, and the Kubernetes controllers client certificate. Enables these components to participate in mutual TLS authentication and identify themselves to the etcd server. | None
+| etcd_cert              | The client certificate issued to `{{site.nodecontainer}}`, the CNI plugin, and the Kubernetes controllers. Enables these components to participate in mutual TLS authentication and identify themselves to the etcd server. | None
```

Comment:
We seem to have dropped the reference to "location of" in the new text. It makes it sound like the config map should contain the actual keys & certs, but I think what is required is the paths to files containing these.